packages/plugin-flex/src/__tests__/commands/flex/plugins/start.test.ts

```diff
@@ -1,7 +1,10 @@
+/* eslint-disable camelcase */
```

Comment:
what's this for?

Reply:
creating the `pluginVersionResource` on line 437 - the interface `PluginVersionResource` has identifiers `account_sid`, `plugin_url`, etc that aren't camel case